### PR TITLE
Refine transnewguinea.org mappings

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

I've gone through and added mappings for more concepts (mainly from recently added concepts)

### CODE REVIEW

**Review by xrotwang — APPROVED** [review]

**Comment by xrotwang** [comment]
```
ERROR:Greenhill-2015-2525: wrong conceptset GLOSS for ID 272: TONG -> TONGS
ERROR:Greenhill-2015-2525:1118: invalid value for concepticon_gloss: TONG
ERROR:Greenhill-2015-2525: wrong conceptset GLOSS for ID 1432: None -> CUT
```

**Comment by LinguList** [comment]
I think we need a proper but quick review here. @AnnikaTjuka, can you moderate it, or review yourself?

**Comment by AnnikaTjuka** [comment]
@LinguList Sure! I'll do a review myself.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 1]
```diff
@@ -1187,27 +1187,27 @@ Greenhill-2015-2525-2491	2491	pipe	1124	PIPE
...
-Greenhill-2015-2525-2509	2509	yam (taitu)		
-Greenhill-2015-2525-2510	2510	to fall (from standing)		
+Greenhill-2015-2525-2502	2502	that (distant)	78	THAT
+Greenhill-2015-2525-2503	2503	there (distant)	1937	THERE
+Greenhill-2015-2525-2504	2504	this (close)	1214	THIS
+Greenhill-2015-2525-2505	2505	this (further)	1214	THIS
+Greenhill-2015-2525-2506	2506	this (lohap)	1214	THIS
+Greenhill-2015-2525-2507	2507	there (lohap)	1937	THERE
+Greenhill-2015-2525-2508	2508	yam (maho)	410	YAM
+Greenhill-2015-2525-2509	2509	yam (taitu)	410	YAM
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Greenhill-2015-2525.tsv:1204]: I'm not sure about the double mapping of the glosses 2503 to 2509. Usually, we map a concept only once in these concept lists.

**Review by AnnikaTjuka — COMMENTED** [review]
I added a few comments. 
@LinguList Some of my comments refer to double mappings. We usually try to avoid double mappings in concept lists but have made exceptions for NoRaRe word lists recently. So I was wondering if it is okay to map a concept twice in the present list.

**Comment by LinguList** [comment]
Double mappings are okay here, I think, sonce TNG is a combination of several concept list, and there is the possibility to split these into individual lists at dome point, when there is time and interest. On the other comments, I cannot comment right now, but largrly support them.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 2]
```diff
@@ -343,9 +343,9 @@ Greenhill-2015-2525-839	839	top	1753	TOP
 Greenhill-2015-2525-840	840	(immediate) imperative plural verb suffix		
 Greenhill-2015-2525-841	841	Locative noun suffix (at, to, from)		
 Greenhill-2015-2525-842	842	to pour out	1487	POUR
-Greenhill-2015-2525-843	843	to buy, sell, barter		
+Greenhill-2015-2525-843	843	to buy, sell, barter	2135	TRADE OR BARTER
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:346]: fixed.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 3]
```diff
@@ -611,15 +611,15 @@ Greenhill-2015-2525-1588	1588	to cast net
...
 Greenhill-2015-2525-1596	1596	citrus fruit sp.	1126	CITRUS FRUIT
 Greenhill-2015-2525-1597	1597	club	1763	CLUB
 Greenhill-2015-2525-1598	1598	coconut mat		
 Greenhill-2015-2525-1600	1600	coral	3179	CORAL ROCK
 Greenhill-2015-2525-1601	1601	corner	192	CORNER
 Greenhill-2015-2525-1602	1602	cousin	1643	COUSIN
-Greenhill-2015-2525-1603	1603	co-wife		
-Greenhill-2015-2525-1604	1604	cowrie		
+Greenhill-2015-2525-1603	1603	co-wife	3636	CO-WIFE
+Greenhill-2015-2525-1604	1604	cowrie	3663	COWRIE SHELL
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:622]: it's the shell.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 4]
```diff
@@ -672,32 +672,32 @@ Greenhill-2015-2525-1693	1693	needlefish sp.
...
 Greenhill-2015-2525-1721	1721	reciprocal action		
 Greenhill-2015-2525-1723	1723	ridge pole	280	RIDGEPOLE
 Greenhill-2015-2525-1726	1726	roast	1962	ROAST (SOMETHING)
 Greenhill-2015-2525-1730	1730	sandfly	95	SANDFLY
 Greenhill-2015-2525-1731	1731	sawfish		
 Greenhill-2015-2525-1736	1736	shadow (shade, shadow)	2275	SHADOW
-Greenhill-2015-2525-1738	1738	sheath		
-Greenhill-2015-2525-1739	1739	shell, hull		
+Greenhill-2015-2525-1738	1738	sheath	3424	SHEATH
+Greenhill-2015-2525-1739	1739	shell, hull	3660	SHELL (SOMETHING)
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:700]: These all look to be nouns, so recoded.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 5]
```diff
@@ -972,13 +972,13 @@ Greenhill-2015-2525-2171	2171	pregnant	1123	PREGNANT
...
 Greenhill-2015-2525-2174	2174	appetiser		
-Greenhill-2015-2525-2175	2175	break (bone)		
+Greenhill-2015-2525-2175	2175	break (bone)	680	BREAK (DESTROY OR GET DESTROYED)
 Greenhill-2015-2525-2176	2176	emphatic pronoun marker		
 Greenhill-2015-2525-2177	2177	greedy, selfish	2018	GREEDY
 Greenhill-2015-2525-2178	2178	pancreas	3252	PANCREAS
 Greenhill-2015-2525-2179	2179	floor	1632	FLOOR
 Greenhill-2015-2525-2180	2180	point		
-Greenhill-2015-2525-2181	2181	sorcery		
+Greenhill-2015-2525-2181	2181	sorcery	392	MAGIC
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:981]: I think they're synonyms here.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 6]
```diff
@@ -989,14 +989,14 @@ Greenhill-2015-2525-2189	2189	2SG (2nd person singular)	1215	THOU
 Greenhill-2015-2525-2190	2190	search	1468	LOOK FOR
 Greenhill-2015-2525-2191	2191	rooster	1511	ROOSTER
 Greenhill-2015-2525-2192	2192	teeth	1380	TOOTH
-Greenhill-2015-2525-2193	2193	meat (game)		
+Greenhill-2015-2525-2193	2193	meat (game)	634	MEAT
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:992]: no -- seems to be a generic term

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 7]
```diff
@@ -1043,24 +1043,24 @@ Greenhill-2015-2525-2244	2244	intransitivizer
...
+Greenhill-2015-2525-2250	2250	newborn	1289	BABY
 Greenhill-2015-2525-2251	2251	to give birth	1195	GIVE BIRTH
 Greenhill-2015-2525-2252	2252	widowed		
-Greenhill-2015-2525-2253	2253	to fetch		
+Greenhill-2015-2525-2253	2253	to fetch	3551	FETCH
 Greenhill-2015-2525-2254	2254	farm	201	FARM
-Greenhill-2015-2525-2255	2255	he, she, it, that, those		
-Greenhill-2015-2525-2256	2256	bad (worthless)		
-Greenhill-2015-2525-2257	2257	(fire) goes out		
+Greenhill-2015-2525-2255	2255	he, she, it, that, those	262	HE OR SHE OR IT
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:1053]: ok, I'll remove this ("HE OR SHE OR IT" seems complicated enough without adding "THAT OR THOSE")

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:1204]: ok - these are Yam cultures so there's lots of types of yam... shall I remove one or both?

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 8]
```diff
@@ -1187,27 +1187,27 @@ Greenhill-2015-2525-2491	2491	pipe	1124	PIPE
...
+Greenhill-2015-2525-2507	2507	there (lohap)	1937	THERE
+Greenhill-2015-2525-2508	2508	yam (maho)	410	YAM
+Greenhill-2015-2525-2509	2509	yam (taitu)	410	YAM
+Greenhill-2015-2525-2510	2510	to fall (from standing)	1280	FALL
 Greenhill-2015-2525-2511	2511	to look for	1468	LOOK FOR
 Greenhill-2015-2525-2512	2512	to break wood		
 Greenhill-2015-2525-2513	2513	woman's clothes (pulpul)		
 Greenhill-2015-2525-2514	2514	to tow		
-Greenhill-2015-2525-2515	2515	young boy		
+Greenhill-2015-2525-2515	2515	young boy	1366	BOY
```
[the remark this answers is not in the document]
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:1210]: yes, two different wordlists in here I think

**Comment by SimonGreenhill** [comment]
Ok -- implemented Annika's comments, except for the one I've left unresolved (YAM).

**Comment by SimonGreenhill** [comment]
...actually, I'll unmap YAM too -- it looks like a species name.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 1 above)
Comment (LinguList) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv]: Maybe better remove both for now.

**Review by AnnikaTjuka — APPROVED** [review]
Many thanks for implementing the changes, @SimonGreenhill! Feel free to merge the PR.

**Comment by AnnikaTjuka** [comment]
I'll merge this PR now.

**Comment by SimonGreenhill** [comment]
thanks!

**Pull request merged**